packages/blob/src/index.ts

```diff
@@ -37,7 +37,8 @@ export interface BlobCommandOptions {
 export interface PutCommandOptions extends BlobCommandOptions {
   access: 'public';
   contentType?: string;
-  handleBlobUploadUrl?: string;
+  handleBlobUploadUrl?: string; // only used in the browser
+  addRandomSuffix?: boolean;
```

Comment:
wonder if we should add the `unique` word in the `addRandomSuffix` flag, to indicate that the flag allows to create a unique link per upload

Reply:
`addRandomSuffix` was the top proposal API name from a ChatGPT conversation :D

I would say it's a fine name